### DISCUSSION TITLE
How to change default width percentage in split view?

### DISCUSSION THREAD

**Discussion by TerryChan** [microsoft/monaco-editor#3733] [Q&A]
Currently the width percentage is 50% - 50% in diff editor with split view by default, how can I change the default width percentage? I want the left view is narrower than the right, is it feasible?

**Comment by TerryChan** [comment]
@alexdima Could you share some information for this issue? I want to change the width percentage by code instead of by dragging the split line by user.

**Comment by TerryChan** [comment]
@alexdima Looks like there is no api to achieve this at current time. Apart from waiting, is there any other workaround to implement this at this period?

**Comment by alexdima** [comment]
@TerryChan The only option is to use editor internals in JS like e.g. `diffEditor._strategy._onSashDrag({ currentX: 20, startX: 100 });`.
